Dockerfile

```diff
@@ -64,16 +64,16 @@ RUN mkdir -p /usr/src
 
 WORKDIR /usr/src/
 
-ADD package.json /usr/src/
-
-ADD yarn.lock /usr/src/
+COPY --from=builder /usr/src/package.json /usr/src/package.json
 
-COPY .yarn /usr/src/.yarn
+COPY --from=builder /usr/src/.yarn /usr/src/.yarn
 
-ADD .yarnrc /usr/src/
+COPY --from=builder /usr/src/.yarnrc /usr/src/.yarnrc
 
 COPY --from=builder /usr/src/packages ./packages
 
+COPY --from=builder /usr/src/yarn.lock /usr/src/yarn.lock
+
```

Comment:
Everything that the production runner uses should be copied from the builder image.